### ISSUE TITLE
Require the admin user to change their password on first login

### ISSUE THREAD

**Issue by jazdw** [infiniteautomation/ma-core-public#1617]
We have the ability to require a user to change their password and a password change page. We should require the admin user to do this the first time they login. We will probably have to look at how to make this work with the license agreement page which usually shows on first login.

We could have a bypass for development mode so we don't annoy our developers with this feature.

**Comment by colbycocking** [comment]
@jazdw Thought I would remind you about this if mango is supposed to be launching next month. Sorry if I am bringing up stuff you are already thinking about.

**Comment by jazdw** [comment]
@colbycocking Yep I am cognizant that there are a lot of things due for Mango 4.0.. Only so many hours in the day :)

@MertCingoz you might be able to take care of this one. I agree with Colby that it is pretty important.
